src/main/java/nextstep/security/csrf/DefaultAccessDeniedHandler.java

```diff
@@ -0,0 +1,13 @@
+package nextstep.security.csrf;
+
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import nextstep.security.authorization.AccessDeniedException;
+import org.springframework.http.HttpStatus;
+
+public class DefaultAccessDeniedHandler implements AccessDeniedHandler {
+    @Override
+    public void handle(final HttpServletRequest request, final HttpServletResponse response, final AccessDeniedException exception) {
+        response.setStatus(HttpStatus.FORBIDDEN.value());
+    }
```

Comment:
Spring Security에서는 응답이 이미 커밋된 경우 추가 처리를 하지 않도록 보호하는 로직을 포함하고 있는데요
이부분도 한번 확인해 보시면 좋을 것 같아요 😄 
```suggestion
    public void handle(final HttpServletRequest request, final HttpServletResponse response, final AccessDeniedException exception) {
    if (response.isCommitted()) {
            return;
    }
        response.setStatus(HttpStatus.FORBIDDEN.value());
    }
```

Reply:
AccessDeniedHandlerImpl 을 확인해보니 에러페이지가 있는 경우 해당 설정도 추가되어있네요! 
좋은 지적 감사합니다